test/services/sync/strategies/TSP/TSP.integration.test.js

```diff
@@ -0,0 +1,75 @@
+const { expect } = require('chai');
```

Comment:
Is it is an integration test?

Reply:
it tests the orchestration of multiple services, so I would call it an integration test